docs/management/connectors/action-types/servicenow.asciidoc

```diff
@@ -1,25 +1,80 @@
 [role="xpack"]
 [[servicenow-action-type]]
-=== ServiceNow connector and action
+=== ServiceNow ITSM connector and action
 ++++
 <titleabbrev>ServiceNow ITSM</titleabbrev>
 ++++
 
-The ServiceNow ITSM connector uses the https://docs.servicenow.com/bundle/orlando-application-development/page/integrate/inbound-rest/concept/c_TableAPI.html[V2 Table API] to create ServiceNow incidents.
+The {sn} ITSM connector uses the https://docs.servicenow.com/bundle/orlando-application-development/page/integrate/inbound-rest/concept/c_TableAPI.html[V2 Table API] to create {sn} incidents.
+
+[float]
+[[servicenow-itsm-connector-prerequisites]]
+==== Prerequisites
+After upgrading to 7.16, you must complete the following within your {sn} instance before creating a new {sn} ITSM connector or <<servicenow-itsm-connector-update, updating an existing one>>:
+
+* Install https://store.servicenow.com/sn_appstore_store.do#!/store/application/7148dbc91bf1f450ced060a7234bcb88[Elastic for ITSM] from the {sn} Store.
+* Create a {sn} integration user and assign it the appropriate roles.
+* Create a Cross-Origin Resource Sharing (CORS) rule in {sn}.
+
+*Create a {sn} integration user*
+
+To ensure authenticated communication between Elastic and {sn}, create a {sn} integration user and assign it the appropriate roles. 
+
+. Go to *System Security -> Users*.
+. Click *New*.
+. Complete the form.
+. Assign the integration user the following roles: 
+* `import_set_loader`
+* `import_transformer`
+* `personalize_choices`
+* `x_elas2_inc_int.integration_user`
+. Click *Submit* to create the user.
+
+*Create a CORS rule*
+
+A CORS rule is required for communication between Elastic and {sn}. To create a CORS rule:
+
+. Go to *System Web Services -> CORS Rules*.
+. Click *New*.
+. Configure the rule settings to use the Elastic ITSM API and the Kibana URL:
+* *REST API*: Choose `Elastic ITSM API [x_elas2_inc_int/elastic_api]`.
+* *Domain*: Enter the Kibana URL.
+. Go to the *HTTP methods* tab and select *GET*. This chooses the GET HTTP method.
+. Click *Submit* to create the rule.
+
+[float]
+[[servicenow-itsm-connector-update]]
+==== Update a deprecated {sn} ITSM connector
+
+{sn} ITSM connectors created in {stack} version <=7.15 are marked as deprecated after you upgrade to >=7.16. Deprecated connectors have a yellow icon after their name and display a warning message when selected.
+
+[role="screenshot"]
+image::management/connectors/images/servicenow-sir-update-connector.png[Shows deprecated ServiceNow connectors]
```

Comment:
The yellow icons are really small in this image; maybe crop it and/or zoom in on the relevant area?

Reply:
@joepeeples retried the magnify tool in Snagnit and made this, wdty?
<img width="1729" alt="servicenow-sir-update-connector-magnified" src="https://user-images.githubusercontent.com/79124755/142250704-4e27c90f-4572-4101-b1b7-c2979818f8d6.png">

Reply:
@nastasha-solomon I like the magnify! Sorry to nitpick, but could you center the icon within the magnified portion?